ur5_demo_descartes/CMakeLists.txt

```diff
@@ -2,7 +2,7 @@ cmake_minimum_required(VERSION 2.8.3)
 project(ur5_demo_descartes)
 
 ## Find catkin macros and libraries
-find_package(catkin REQUIRED descartes_core descartes_moveit pluginlib moveit_core roscpp tf_conversions)
+find_package(catkin REQUIRED descartes_core descartes_moveit pluginlib moveit_core roscpp tf_conversions ur_kinematics)
```

Comment:
My error could've been caused by interaction between what is described in ros-industrial/universal_robot#217 and user code.

Sorry for the noise (but 217 really needs to be fixed :)).